### PR TITLE
add styling to suggestions

### PR DESCRIPTION
**Pull request by Sarah-Addo**
Screenshot: https://screenshot.googleplex.com/XtLUT7KHszY
The site has also been deployed for you to view. Currently overflow is taken care of with css but will try to find another way to load comments?

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/AddCommentServlet.java [hunk 1]
```diff
@@ -36,11 +36,13 @@
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     // Get the input from the form.
+    String newName = request.getParameter("name") + ":";
```
Comment (belraquib) [inline portfolio/src/main/java/com/google/sps/servlets/AddCommentServlet.java:39]: What happens if this is null or empty?

File: portfolio/src/main/webapp/suggestions_script.js [hunk 2]
```diff
@@ -11,37 +11,45 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
 document.addEventListener("DOMContentLoaded", async function() {
     let response = await fetch('/list-comments');
     let comments = await response.json();
+    console.log(comments);
```
Comment (belraquib) [inline portfolio/src/main/webapp/suggestions_script.js:18]: remove your logs :)

File: portfolio/src/main/webapp/suggestions_script.js [hunk 3]
```diff
@@ -11,37 +11,45 @@
...
   });
-
+ 
 function createComment(comment) {
-  const commentElement = document.createElement('li');
-  commentElement.className = 'comment';
-
-  const commentTextElement = document.createElement('span');
+  const commentElement = document.createElement("div");
+  commentElement.setAttribute("id", "comment");
```
Comment (belraquib) [inline portfolio/src/main/webapp/suggestions_script.js:28]: This gives each created comment an id of comment. perhaps you want to set this as a class instead?

commentElement.classList.add('comment');

File: portfolio/src/main/webapp/suggestions_script.js [hunk 4]
```diff
@@ -11,37 +11,45 @@
...
+ 
+  const commentNameElement = document.createElement('div');
+  commentNameElement.setAttribute("id", "comment-name");
```
Comment (belraquib) [inline portfolio/src/main/webapp/suggestions_script.js:31]: same as above. I think you want comment-name to be a class.

File: portfolio/src/main/webapp/suggestions_script.js [hunk 5]
```diff
@@ -11,37 +11,45 @@
...
+  const commentTextElement = document.createElement('div');
+  commentTextElement.setAttribute("id", "comment-text");
   commentTextElement.textContent = comment.commentText;
-
-  const deleteButtonElement = document.createElement('button');
-  deleteButtonElement.textContent = 'Delete';
+ 
+  const deleteButtonElement = document.createElement('div');
+  deleteButtonElement.setAttribute("id", "delete-button");
+  deleteButtonElement.innerText = 'X';
```
Comment (belraquib) [inline portfolio/src/main/webapp/suggestions_script.js:39]: textcontent

File: portfolio/src/main/java/com/google/sps/servlets/AddCommentServlet.java (hunk 1 above)
Comment (Sarah-Addo) [reply portfolio/src/main/java/com/google/sps/servlets/AddCommentServlet.java:39]: Now will set name to anonymous

**Pull request merged**